src/components/UI/Controls/VersionPicker/VersionPicker.tsx

```diff
@@ -160,13 +160,6 @@ const VersionPicker: React.FC<IVersionPickerProps> = ({
     setIncludeTestVersions(event.target.checked);
   };
 
-  const handleBlur = (onBlurHandler: () => void) => () => {
-    setTimeout(() => {
-      onBlurHandler();
-      // eslint-disable-next-line no-magic-numbers
-    }, 300);
-  };
```

Comment:
No idea what this was solving, but whatever the problem was, it doesn't happen anymore